src/layouts/partial-cards.html

```diff
@@ -253,15 +276,16 @@
     return window.hassUtil.isComponentLoaded(hass, 'introduction');
   },
 
-  computeViews: function (hass) {
+  computeAllViews: function (hass) {
     var views = window.HAWS.extractViews(hass.states);
-
     // If default view present, it's in first index.
-    if (views.length > 0 && views[0].entity_id === 'group.default_view') {
-      views.shift();
+    if (views.length > 0 && views[0].entity_id === this.DEFAULT_VIEW_ENTITY_ID) {
+      this.defaultView = views.shift();
+    } else {
+      this.defaultView = null;
     }
 
-    return views;
+    this.views = views;
```

Comment:
Instead of having this function being an observer with a side-effect, make it a computed property. Example: https://github.com/home-assistant/home-assistant-polymer/blob/master/panels/config/ha-panel-config.html#L100